schema.graphql

```diff
@@ -90,15 +90,9 @@ type Activity @entity {
   type: ActivityType!
   timestamp: BigInt!
 
-  # mint activity
-  mintBy: User
-
-  # burn activity
-  burnBy: User
-
-  # transfer activity
-  transferFrom: User
-  transferTo: User
+  actor: User
```

Comment:
I just suggest calling this `sender` to keep it consistent with near contract `context.sender`

Reply:
@achrafismyname I think, we should add descriptions for the entities. Then It will be easier to understand.